locale/fi/LC_MESSAGES/django.po

```diff
@@ -670,23 +670,15 @@ msgstr "Ei vielä ryhmiä"
 msgid "My courses"
 msgstr "Minun kurssini"
 
-#: course/templates/course/index.html:57
+#: course/templates/course/index.html:57 course/templates/course/index.html:136
 msgid "View course"
 msgstr "Näytä kurssi"
 
 #: course/templates/course/index.html:71
-msgid "Kaikki kurssit"
-msgstr "All courses"
-
-#: course/templates/course/index.html:90
-msgid "Lue lisää"
-msgstr "Read more"
-
-#: course/templates/course/index.html:136
-msgid "View course"
-msgstr "Näytä kurssi"
+msgid "All courses"
```

Comment:
Should this translation be part of previous commits?

Reply:
Probably yes, I will correct that

Reply:
Translations are in the part where they belong now